### PR TITLE
Update Signup requests tab in real-time

### PR DESCRIPTION
**Pull request by padamchopra**
The entire Bassa site works on a real-time basis which makes the user experience a whole lot better. This had not been implemented in the area of the admin approving a signup request from a new user. 
Now, when the admin approves a signup request, the request disappears from the page and hence it is updated in real-time

View the working demonstration of this [here](https://drive.google.com/file/d/1CcO7PfMnlY6CoaEFlVIQP8b04yse7U6q/view?usp=sharing)

### CODE REVIEW

**Comment by padamchopra** [comment]
@JaDogg @agentmilindu @charitha-scorelab 
Can anyone review this? Its just a minor addition of code which will make the signup requests tab update in real-time, thus making the overall user experience more sweet :clinking_glasses:

File: ui/src/app/views/admin.html [hunk 1]
```diff
@@ -18,12 +18,12 @@
...
     <table id="table" class="table table-hover table-bordered signup-requests-table">
         <tbody>
-        <tr ng-repeat="data in signup_requests">
+        <tr ng-repeat="data in signup_requests" ng-hide="number_to_approve == $index">
             <td data-title="ID">{{$index + 1}}</td>
             <td data-title="Name">{{data.user_name}}</td>
             <td data-title="Email">{{data.email}}</td>
             <td id="approve-btn">
-              <md-button class="md-raised" ng-click="approve(data.user_name)">
+              <md-button class="md-raised" ng-click="approve(data.user_name); number_to_approve = $index">
```
Comment (malithsen) [inline ui/src/app/views/admin.html]: Having logic in both view and controller will make the code hard to debug and maintain. Let's move this inside the controller

Comment (padamchopra) [reply ui/src/app/views/admin.html]: I've shifted the logic to the controller now

**Comment by agentmilindu** [comment]
@malithsen @JaDogg Please review and merge the updated PR.

File: ui/src/app/admin/controllers/AdminCtrl.js [hunk 2]
```diff
@@ -46,8 +46,9 @@
     };
 
     $scope.approve = function(username) {
-      AdminService.approve(username).then(function (response) {
-        ToastService.showToast('Approved', username);
+      AdminService.approve(username.user_name).then(function (response) {
```
Comment (agentmilindu) [inline ui/src/app/admin/controllers/AdminCtrl.js]: `username.user_name` looks ugly, shall we change the param to `user` and get as `user.user_name`?

Comment (padamchopra) [reply ui/src/app/admin/controllers/AdminCtrl.js]: Done. I'll be a little more careful from the next time with variable and parameter names :sweat_smile:

**Comment by padamchopra** [comment]
@agentmilindu @malithsen Changes have been made. Please review and merge if appropriate

**Comment by malithsen** [comment]
@padamchopra One of the test cases fails now possibly due to https://github.com/scorelab/Bassa/pull/422/commits/07e7d30ca8f064b97bed341f6cc530e588938201

**Comment by malithsen** [comment]
Updating the test case to match the variable name change should fix it

**Comment by padamchopra** [comment]
@malithsen It works for me in all possible cases. Can you elaborate on the change you need? 😟

**Comment by malithsen** [comment]
@padamchopra Travis build fails because there's a failing test case. https://travis-ci.org/scorelab/Bassa/builds/339751459#L1325

This could be due to the variable name change in one of the previous commits.

File: ui/src/app/admin/controllers/AdminCtrl.spec.js [hunk 3]
```diff
@@ -50,6 +50,13 @@ describe('Controller: Admin controller', function() {
...
     spyOn(AdminService, 'approve').and.callFake(fakeHttpPromise);
   
+
+    var user = {
+        user_name: 'BassaUser',
+        email: 'user@bassa.com',
+        password: 'bassapass'
+    }
+
     scope.approve();
```
Comment (malithsen) [inline ui/src/app/admin/controllers/AdminCtrl.spec.js]: You need to pass the user object to the approve function
`scope.approve(user);`

Comment (padamchopra) [reply ui/src/app/admin/controllers/AdminCtrl.spec.js]: Done

**Comment by malithsen** [comment]
Good work @padamchopra :+1:

**Pull request merged**